app/assets/src/components/visualizations/GenomeViz.js

```diff
@@ -142,8 +150,36 @@ export default class GenomeViz {
   onMouseLeave = () => {
     if (this.options.onGenomeVizBarExit) {
       this.options.onGenomeVizBarExit();
-      this.lastHoveredDataIndex = null;
     }
+    this.unhighlightBar(this.lastHoveredDataIndex);
+    this.lastHoveredDataIndex = null;
+  };
+
+  highlightBar = barIndex => {
+    if (barIndex === null) {
+      return;
+    }
+    const colors = this.getColors();
+
+    this.svg
+      .select(`.bar-container .rect-${barIndex}`)
+      .attr(
+        "fill",
+        color(colors[this.data[barIndex][2]]).darker(
+          this.options.hoverDarkenFactor
+        )
+      );
+  };
+
+  unhighlightBar = barIndex => {
```

Comment:
seems like the only diff with method above is `.darker(this.options.hoverDarkenFactor)` ... refactor?